README.md

```diff
@@ -10,18 +10,19 @@ will always override the value provided in the yaml file.
 
 ### Environment variables
 
-| Name                        | Description                              | Type     | Default                                |
-|-----------------------------|------------------------------------------|----------|----------------------------------------|
-| INFINITEWISDOM_BOT_TOKEN                   | The bot token used to authenticate the bot with telegram | String | `-` |
-| INFINITEWISDOM_MAX_URL_POOL_SIZE           | Maximum number of URLs to keep in the pool | Integer | `10000` |
-| INFINITEWISDOM_IMAGE_POLLING_TIMEOUT       | Timeout in seconds between image api requests | Integer | `1` |
-| INFINITEWISDOM_GREETING_MESSAGE            | Specifies the message a new user is greeted with | String | `Send /inspire for more inspiration :) Or use @InfiniteWisdomBot in a group chat and select one of the suggestions.` |
-| INFINITEWISDOM_INLINE_BADGE_SIZE           | Number of items to return in a single inline request badge | Integer | `16` |
-| INFINITEWISDOM_PERSISTENCE_TYPE            | Type of persistence to use | String | `local` |
-| INFINITEWISDOM_PERSISTENCE_PATH            | Path of pickle persistence | String | `/tmp/infinitewisdom.pickle` |
-| INFINITEWISDOM_PERSISTENCE_URL             | URL to use for SQLAlchemy connection | String | `sqlite:////tmp/infinitewisdom.db` |
-| INFINITEWISDOM_IMAGE_ANALYSIS_TYPE         | Type of image analysis to use | String | `None` |
-| INFINITEWISDOM_IMAGE_ANALYSIS_AUTH_FILE    | Path of Google Vision auth file | String | `None` |
+| Name                                                  | Description                              | Type     | Default                                |
+|-------------------------------------------------------|------------------------------------------|----------|----------------------------------------|
+| INFINITEWISDOM_TELEGRAM_BOT_TOKEN                     | The bot token used to authenticate the bot with telegram | `str` | `-` |
+| INFINITEWISDOM_TELEGRAM_GREETING_MESSAGE              | Specifies the message a new user is greeted with | `str` | `Send /inspire for more inspiration :) Or use @InfiniteWisdomBot in a group chat and select one of the suggestions.` |
+| INFINITEWISDOM_TELEGRAM_INLINE_BADGE_SIZE             | Number of items to return in a single inline request badge | `int` | `16` |
+| INFINITEWISDOM_CRAWLER_TIMEOUT                        | Timeout in seconds between image api requests | `float` | `1` |
```

Comment:
Isn't it more like an `interval`? `timeout` sounds like "how many seconds will the crawler wait for a response".
(Yes I know it was already called timeout before :wink:)

Reply:
I also didn't like the term *timeout* but I stuck with it because in the code it is not used as a fixed interval. It's the time to wait after the previous task has finished before starting a new one. Maybe this should just be changed to make it a real interval.

Reply:
Scheduling the next run before the current is finished might lead to problems though if more tasks are scheduled than are processed. It will pile up indefinitely. I will just change the name, a more sophisticated solution to this problem can be implemented later.

Reply:
>Scheduling the next run before the current is finished might lead to problems though if more tasks are scheduled than are processed. It will pile up indefinitely. I will just change the name, a more sophisticated solution to this problem can be implemented later.

I was just writing a similar explanation :)